567_leetcode_Permutation_in_String.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def checkInclusion(self, s1: str, s2: str) -> bool:
+        if len(s1) > len(s2):
```

Comment:
Assign constant operation value to single variable.
Example: s1Length = len(s1)